packages/core/src/RenderingEngine/VideoViewport.ts

```diff
@@ -445,6 +485,24 @@ class VideoViewport extends Viewport implements IVideoViewport {
     };
   }
 
+  public hasImageURI(imageURI: string) {
```

Comment:
also please add documentation, on what is matched and what is not (I guess frame range is matched)